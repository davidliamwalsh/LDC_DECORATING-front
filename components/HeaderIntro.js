import { Component } from 'react'
import Link from 'next/link'

class HeaderIntro extends Component {
  
  render () {
    return <div className="c-top" id="head-intro">
      <div className="c-header-intro">
        <div className="c-header-intro__container">
          <div className="c-header-intro__contact">
            <li className="c-header-intro__contact--inset">
              <Link href='tel:03303201070'>
                <a>03303 201070</a>
              </Link>
            </li>
            <li>|</li>
            <li className="c-header-intro__contact--inset">
              <Link href='mailto:info@ldcdecoratingcontractors.co.uk'>
                <a>info@ldcdecoratingcontractors.co.uk</a>
              </Link>
            </li>
          </div>
          <div className="c-header-intro__links">
            <li className="c-header-intro__links--link">
              <Link href='/services'>
                <a>Services</a>
              </Link>
            </li>
            <li>|</li>
            <li className="c-header-intro__links--link">
              <Link href='/contact'>
                <a>Contact</a>
              </Link>
            </li>
          </div>
        </div>
      </div>
    </div>
  }
}

export default HeaderIntro