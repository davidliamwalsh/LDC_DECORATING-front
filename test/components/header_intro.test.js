import HeaderIntro from '../../components/HeaderIntro'

describe('<HeaderIntro />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <HeaderIntro />
    )

    expect(component).toMatchSnapshot()
  })
})