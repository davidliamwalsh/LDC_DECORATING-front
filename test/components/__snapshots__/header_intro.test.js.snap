// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderIntro /> it renders correctly 1`] = `ShallowWrapper {}`;
